HoloStorageAccessor/internal/openapi/routers.go

```diff
@@ -0,0 +1,161 @@
+/*
+ * HoloStorage Accessor API
+ *
+ * API to access holograms and metadata from HoloStorage
+ *
+ * API version: 1.0.0
+ * Generated by: OpenAPI Generator (https://openapi-generator.tech)
+ */
+
+package openapi
+
+import (
+	"log"
+	"net/http"
+
+	"github.com/gin-gonic/gin"
+)
+
+// Route is the information for every URI.
+type Route struct {
+	// Name is the name of this Route.
+	Name string
+	// Method is the string for the HTTP method. ex) GET, POST etc..
+	Method string
+	// Pattern is the pattern of the URI.
+	Pattern string
+	// HandlerFunc is the handler function of this route.
+	HandlerFunc gin.HandlerFunc
+}
+
+// Routes is the list of the generated Route.
+type Routes []Route
+
+type AccessorConfig struct {
+	FhirURL       string `json:"fhirUrl"`
+	BlobStoreName string `json:"blobStoreName"`
+	BlobKey       string `json:"blobKey"`
+}
+
+var accessorConfig AccessorConfig
+
+// NewRouter returns a new router.
+func NewRouter() *gin.Engine {
+	err := LoadConfiguration(&accessorConfig)
+	if err != nil {
+		log.Fatalln(err)
+	}
+
+	log.Printf("Fhir URL: %q", accessorConfig.FhirURL)
+	log.Printf("Blob Store: %q", accessorConfig.BlobStoreName)
+
+	router := gin.Default()
+
+	router.Static("/app/1.0.0/ui/", "./third_party/swaggerui")
+
+	for _, route := range routes {
+		switch route.Method {
+		case http.MethodGet:
+			router.GET(route.Pattern, route.HandlerFunc)
+		case http.MethodPost:
+			router.POST(route.Pattern, route.HandlerFunc)
+		case http.MethodPut:
+			router.PUT(route.Pattern, route.HandlerFunc)
+		case http.MethodDelete:
+			router.DELETE(route.Pattern, route.HandlerFunc)
+		}
+	}
+
+	return router
+}
+
+// Index is the index handler.
+func Index(c *gin.Context) {
+	c.String(http.StatusOK, "Hello World!")
+}
+
+var routes = Routes{
+	{
+		"Index",
+		http.MethodGet,
+		"/api/1.0.0/",
```

Comment:
Maybe the prefix `"/api/1.0.0/"` (or even just the `1.0.0`) could be stored in a variable and added below with string interpolation (or just concatenation), so that a version change won't require many lines to change

Reply:
Shifted prefix to a common variable